XLS-0033d-compact-fungible-tokens/README.md

```diff
@@ -206,122 +216,28 @@ A set of flags indicating properties or other options associated with this **`CF
-
-###### 1.2.1.3.1.3. `**NextPageMin**` 
-
-The locator of the next page, if any. Details about this field and how it should be used are outlined below, after the construction of the **`CFTokenPageID`** is explained.
-
-###### 1.2.1.3.1.4. `**PreviousTxnID**` 
+###### 1.2.1.2.1.7. `OwnerNode`
 
-Identifies the transaction ID of the transaction that most recently modified this **`CFTokenPage`** object.
-
-###### 1.2.1.3.1.5. `**PreviousTxnLgrSeq**` 
-
-The sequence of the ledger that contains the transaction that most recently modified this **`CFTokenPage`** object.
-
-###### 1.2.1.3.1.6. `**CFTokens**` 
-
-The collection of **`CFToken`** objects contained in this **`CFTokenPage`** object. This specification places an upper bound of 32 **`CFToken`** objects per page. Objects should be stored in sorted order, from low to high with the low order 96 bits of the `TokenID` used as the sorting parameter.
-
-##### 1.2.1.3.2. CFTokenPage ID Format
-
-Unlike other object identifiers on the XRP Ledger, which are derived by hashing a collection of data using `SHA512-Half`, **`CFTokenPage`** identifiers are constructed so as to specfically allow for the adoption of a more efficient paging structure, designed to enable user accounts to efficiently hold many CFTs at the same time.
-
-To that end, a unique CFTokenPage ID (a.k.a., `CFTokenPageID`) is derived by concatenating a 196-bit value that uniquely identifies a particular account's holdings of CFT, followed by a 64-bit value that uniquely identifies a particular CFT issuance. Using this construction enables efficient lookups of individual `CFTokenPage` objects without requiring iteration of the doubly-linked list of all CFTokenPages.
-
-More formally, we assume:
-
-- The function `high196(x)` returns the "high" 196 bits of a 256-bit value.
-- The function `low64(x)` returns the "low" 64-bits of a 256-bit value. 
-- A `CFTokenIssuanceID` uniqely identifies a CFT Issuance as defined above in ["CFTokenIssuance Ledger Identifier"].
-- A `CFTokenHolderID` uniquely identifies a holder of some amount of CFT (as opposed to other token types such as an NFT) and is defined as the result of SHA512-Half of the following values, concatenated in order:
-  - The `CFTokenIssuance` ledger identifier key (0x007E).
-  - The `AccountID` of the CFT holder.
-
-Therefore:
+Identifies the page in the owner's directory where this item is referenced.
 
-- Let `CFTokenPageID` equal `high196(CFTokenHolderId)` concatenated with `low64(CFTokenIssuanceId)`.
-- Let `CFTokenIssuanceID` `A` only be included in a page with `CFTokenPageId` `B` if and only if `low64(A) >= low64(B)`.
+###### 1.2.1.2.1.8. `CFTokenNode`
 
-This scheme is similar to the existing scheme for organizing `NFToken` objects into `NFTokenPage`s.
+Identifies the page in the CFT directory where this item is referenced.
```

Comment:
What is the `CFT Directory` that you're referring to? Are you proposing a new object here?

Currently, there are only two types of [Owner Directory[(https://xrpl.org/directorynode.html#directorynode) nodes -- one for `offers` and one for `everything else`.

My understanding from our last discussion was that `CFTokenIssuance` and `CFToken` would both live in their owner's owner dirctories (the "everything else" kind, above).

Reply:
I'll make a separate PR to discuss this one.